docs/application/vscode-ext/web.md

```diff
@@ -0,0 +1,306 @@
+
+1. Open **Command Palette** and enter **Run Emulator-Manager**. The Tizen Emulator Manager is launched.
+
+   > **Note**  
+   > You cannot launch an emulator directly from VS Code. Instead, you must launch the Tizen Emulator Manager and use it to launch the emulator you need.
+
+2. Create and launch an emulator instance in the Emulator Manager.
+3. To deploy your application to the target, enter **Install Tizen App** in the **Command Palette**.
+4. To run the application on the emulator, enter **Run Tizen App** in the **Command Palette**.
+
+To install your project:
+
+  1. Open **Command Palette** and enter **Install Tizen App**.
+  2. Enter the folder name of the project you want to install. (Folder name, not path)
+  3. When the installation is complete, it succeeds, and there is successful generation of the installation app. Also, you will see a build success log in the **terminal** at the bottom on the vscode.
+
+  ![Install your tizen app](media/tizen_web14.png)
+
+> **Note** 
+> 
+> If you have a Tizen device and you want to deploy the application to it, you must first generate, and then add a certificate.
+
+To run your project:
+
+  1. Open **Command Palette** and enter **Run Tizen App**.
+  2. Enter the folder name of the project you want to run. (Folder name, not path)
+  3. When the run is complete, it succeeds, and there is successful generation of launch app popup. Also, you will see a build success log in the terminal at the bottom on the vscode.
+
+  ![Run Your Tizen App](media/tizen_web15.png)
+
+> **Note**  
+>
+> If you have a Tizen device and you want to deploy the application to it, you must first generate, and then add a certificate.
+>
+> In addition, vscode terminal displays successful launch log messages:
+> 
+> 1. debug 0 means normal launch.
+> 2. debug 1 means launching an Inspector available.
+
+### Debug Application in Emulator
+
+To debug application, follow these steps:
+
+> **Note**
+>
+> Ensure the following:
+> - Emulator is launched
+> - Device is connected to the PC
+> - Chrome browser is installed
+
+1. Open **Command Palette** and enter **Debug Tizen App**.
+2. Enter the name of the Tizen Web App you want to run.
+3. A successful popup is created and the app is launched.
+
+> **Note**
+> 
+> In addition, `vscode` terminal displays successful launch log messages:
+> vscode Terminal displays a successful launch log
+>
+> - debug `0` means normal launch.
+> - debug `1` means launching an inspector available from command line.
+> - Additionally, debug the connected port as shown in the following log.
+
+  ![Debugging you application](media/tizen_web16.png)
+
+4. Connect **Debug Tizen App** with **Inspector**.
+5. Open **Command Palette** and enter **Run Inspector**.
+6. Enter the port number on your local PC, for example, 9999
+7. In Chrome browser, the Inspector automatically launches and associates with the launched app.
+
+    ![Debugging you application](media/tizen_web17.png)
```

Comment:
@sabeena21 
It seems to be due to the color of my monitor.
Do you need to prepare a new image on another PC?

Reply:
@DohyungLim Yes, I think it would be better to use the normal color based screenshot.
Would you please update the image? Also please don't forget to remove personal information.

Reply:
We're going to update the image.
You must prepare the image first. Therefore, it will not be applied to the current patch.
I'm going to update it during the review.